src/io/flutter/run/LauncherState.java

```diff
@@ -50,28 +50,28 @@
 
 /**
  * Launches a flutter app, showing it in the console.
- *
- * <p>Normally creates a debugging session, which is needed for hot reload.
+ * <p>
+ * Normally creates a debugging session, which is needed for hot reload.
  */
-public class Launcher extends CommandLineState {
+public class LauncherState extends CommandLineState {
```

Comment:
The `Launcher` name seemed a bit short to me; all of the IntelliJ subclasses of `RunProfileState` are suffixed with `State`. I prefer `LauncherState` to just `Launcher`, but will leave it as `Launcher` if you feel strongly.